### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by lisandroseia**
Hi, and thanks for the review!
In this branch, I created migrations tables following this [diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)
Please review and let me know if there is something wrong!

### CODE REVIEW

File: db/migrate/20220428211016_add_user_ref_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (rloterh) [inline db/migrate/20220428211016_add_user_ref_to_posts.rb]: - [ ] As per the given ERD, the _Author_ model is acting as a through model association to _User_ model, hence implementing a foreign key by creating _author_id_ and a corresponding _index_ should be similar to:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Kindly ensure to improve upon this aspect.

File: db/migrate/20220428211227_add_user_ref_to_likes.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```
Comment (rloterh) [inline db/migrate/20220428211227_add_user_ref_to_likes.rb]: - [x] As per the given ERD, the _Author_ model is acting as a through model association to _User_ model, hence implementing a foreign key by creating _author_id_ and a corresponding _index_ should be similar to:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Kindly ensure to improve upon this aspect.

File: db/migrate/20220428214622_add_user_ref_to_comments.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```
Comment (rloterh) [inline db/migrate/20220428214622_add_user_ref_to_comments.rb]: - [x] As per the given ERD, the _Author_ model is acting as a through model association to _User_ model, hence implementing a foreign key by creating _author_id_ and a corresponding _index_ should be similar to:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Kindly ensure to improve upon this aspect.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Lisandro, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
Hi @lisandroseia , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**